inc/class-importer.php

```diff
@@ -311,11 +322,71 @@ protected function update_post_from_markdown_source( $post_id ) {
 		if ( ! is_null( $title ) ) {
 			$post_data['post_title'] = sanitize_text_field( wp_slash( $title ) );
 		}
+		if ( isset( $yaml['published'] ) ) {
+			$post_data['post_status'] = (bool) $yaml['published'] ? 'publish' : 'draft';
+		}
+		if ( isset( $yaml['date'] ) ) {
+			$post_data['post_date'] = date( 'Y-m-d H:i:s', strtotime( $yaml['date'] ) );
+		}
+		if ( isset( $yaml['author'] ) ) {
```

Comment:
What about securities?

Reply:
I guess it wouldn't really change things that much - if a user is found they would still only be able to make changes if their user role for the site allowed it. Not too familiar with how things are set up on wp.org.

Reply:
Given changes go through a review cycle and that it's totally optional to add that I don't think it's a big problem personally.